pythonforandroid/recipes/pydantic/__init__.py

```diff
@@ -0,0 +1,12 @@
+from pythonforandroid.recipe import PythonRecipe
+
+
+class PydanticRecipe(PythonRecipe):
+    version = '1.8.2'
+    url = 'https://github.com/samuelcolvin/pydantic/archive/refs/tags/v{version}.zip'
+    depends = ['setuptools']
+    python_depends = ['Cython', 'devtools', 'email-validator','dataclasses', 'typing-extensions', 'python-dotenv']
```

Comment:
Minor linter issue here it seems:
```
pythonforandroid/recipes/pydantic/__init__.py:8:62: E231 missing whitespace after ','
```
Let's fix it so the pipeline runs fully

Reply:
> Also if you know how to squash your commits so your PR is only one commit perfect, otherwise we'll try to think about it upon merge
I am searching everywhere here in the Github interface and I can't find where to squash my commits. 
Is it possible to do it from github.com interface or do I need to clone the repo and so on?

Reply:
Thanks for looking into it.
I'm not aware of how to do it from GitHub.
I usually do it from the command line using either `git rebase -i...` or `git reset --soft HEAD~...`.
I'll squash upon merge then